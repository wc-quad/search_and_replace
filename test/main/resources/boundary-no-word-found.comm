R lion flee
R tiger what
R eat flew
Q
