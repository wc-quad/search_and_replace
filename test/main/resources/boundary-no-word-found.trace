Give a sequence of commands of the form:
s word             [search]
r word replacement [replace]
Type 'q' to quit
