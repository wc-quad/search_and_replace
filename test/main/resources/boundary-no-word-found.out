A flea and a fly in a flue
Were imprisoned, so what could they do?
Said the fly, "let us flee!"
"Let us fly!" said the flea.
So they flew through a flaw in the flue.
